### PR TITLE
DISPATCH-1719 logging and vhost note

### PR DESCRIPTION
**Pull request by pwright**

### CODE REVIEW

**Comment by pwright** [comment]
@ted-ross does this address the issue?

File: docs/books/modules/user-guide/viewing-log-entries.adoc [hunk 1]
```diff
@@ -50,6 +50,8 @@ Wed Jun  7 17:49:32 2019 SERVER (none) [2]:0 -> @flow(19) [next-incoming-id=1, i
 ----
 --
 
+NOTE: `vhost` entries are only logged if `multiTenant` is set to `true` in the router configuration file.
+
```
Comment (bhardesty) [inline docs/books/modules/user-guide/viewing-log-entries.adoc:54]: FYI, I don't know if it's necessary in this case, but if you want to refer to the default path and name of the config file, you can use this attribute:

https://github.com/apache/qpid-dispatch/blob/df22f36b9d45ccf9c555f847919c911e6c6d57e8/docs/books/common/attributes.adoc#L46

**Comment by gemmellr** [comment]
Please use DISPATCH Jira references.

**Comment by pwright** [comment]
@fgiorgetti  could you approve this?

**Comment by fgiorgetti** [comment]
@pwright looks good to me,

**Comment by pwright** [comment]
@fgiorgetti  I'm not sure who should merge this, I don't have the privs

**Comment by jiridanek** [comment]
@bhardesty has the powers. Or me. I'll merge this myself now.

**Pull request merged**